code/game/mecha/combat/gygax.dm

```diff
@@ -67,6 +67,15 @@
 	desc = "A light security exosuit originally manufactured by Cybersun Biodynamics, extensively modified by IRMG artificers. The leg actuators have been maxed out, allowing for powerful short ranged charges capable of breaking walls and other obstacles."
 	icon_state = "inteqgygax"
 
+/obj/mecha/combat/gygax/charger/overclock
+	name = "\improper Overclocked 501p"
+	desc = "A lightweight security exosuit, which has been overclocked to have it's leg actuators launch the exosuit forward instead of enhanced speed."
```

Comment:
```suggestion
	desc = "A lightweight security exosuit, which has been overclocked to have its leg actuators launch the exosuit forward instead of enhanced speed."
```